### ISSUE TITLE
Should we get the orchestrator log output in the Unity log for the session creator?

### ISSUE THREAD

**Issue by jackjansen** [cwi-dis/VR2Gather#154]
@troeggla assigning this to you for initial feedback.

I'm thinking that it would be a good idea to get the orchestrator log output into the Unity log, for at least one of the users. The best candidate is probably the session creator.

The implementation would be rather simple: there would be a completely independent thread in VR2Gather that would use the orchestrator log-reading interface, and it would simply dump anything it gets into the Unity log. So, it would contain all orchestrator log messages, including those for other sessions and so. Especially those, I might say, because they could pinpoint obscure problems where different sessions somehow influence each other.

First question: is this feasible from the orchestrator side? Can there be multiple readers of the log?

Second question: what is the orchestrator API for reading the log files? Is it simple streamed HTTP or is it socketIO?

Third question: or is this a stupid idea?

**Comment by troeggla** [comment]
Yes, there can be any number of readers of the log. It is implemented as a Socket.IO endpoint named `log` under the namespace `/log`, that will fan out anything dumped into it to all listeners
